cmake/kokkos_functions.cmake

```diff
@@ -347,7 +347,7 @@ macro(kokkos_create_imported_tpl NAME)
       target_link_libraries(${NAME} INTERFACE ${TPL_LINK_LIBRARIES})
     endif()
     if(TPL_INCLUDES)
-      target_include_directories(${NAME} INTERFACE ${TPL_INCLUDES})
+      target_include_directories(${NAME} SYSTEM INTERFACE ${TPL_INCLUDES})
```

Comment:
@dalg24 This change was introduce in an attempt to silence some `Thrust` warnings, did not work. Should I revert ?

Reply:
I am not sure it's a good idea for us to inject "system" things into stuff downstream users get. Is this doing that?

Reply:
@crtrott Indeed, this does not solve the problem of the `thrust` related warnings. I'll revert.

Reply:
Done, closing :heavy_check_mark: